lib/event.rb

```diff
@@ -0,0 +1,55 @@
+require './lib/person'
+require './lib/craft'
+
+class Event
+  attr_reader :name,
+              :crafts,
+              :attendees
+
+  def initialize(name, crafts, attendees)
+    @name = name
+    @crafts = crafts
+    @attendees = attendees
+  end
+
+  def attendee_names
+    @attendees.map do |attendee|
+      attendee.name
+    end
+  end
+
+  def craft_with_most_supplies
+    most = @crafts.max_by do |craft|
+      craft.supplies_required.length
+    end
+    most.name
+  end
+
+  def supply_list
+    string = @crafts.map do |craft|
```

Comment:
What is this actually returning here? I think it is an array of supplies as symbols? So what might a better variable name be instead of `string`?